app/assets/stylesheets/barnardos/_reactive-preview.scss

```diff
@@ -0,0 +1,26 @@
+.reactive-preview {
+  position: relative;
+}
+
+.reactive-preview__section {
+  background: whitesmoke;
+  font: 16px/24px;
+  padding: 8px 16px;
+}
+
+.reactive-preview__section:not(:first-child) {
+  margin-top: 8px;
+}
+
+.reactive-preview__highlight {
+  background: moccasin;
+  word-break: break-word;
```

Comment:
@joelcarr RubyMine is telling me `break-word` is not a valid value for this. Is RubyMine right?

Reply:
Here is a list of options for `word-break` available in Chrome: 
![image](https://user-images.githubusercontent.com/893208/33374625-d8c7f980-d4fe-11e7-92b0-95ef74575b54.png)